src/visualize_paths.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+//jscs:disable requireSpacesInForStatement
+//jscs:disable requireSpaceBeforeBinaryOperators
+//jscs:disable disallowMultipleVarDecl
+//jscs:disable validateQuoteMarks
+//jscs:disable requireSpaceBeforeObjectValues
+//jscs:disable validateIndentation
+//jscs:disable requireCurlyBraces
+//jscs:disable requireSpaceAfterKeywords
+//jscs:disable requireSpaceBetweenArguments
+//jscs:disable requireSpacesInConditionalExpression
+//jscs:disable requireSpaceAfterBinaryOperators
+//jscs:disable disallowSpaceAfterPrefixUnaryOperators
+//jscs:disable validateParameterSeparator
+//jscs:disable requireSpaceBeforeBlockStatements
+//jscs:disable disallowSpacesInsideObjectBrackets
+//jscs:disable disallowSpacesInAnonymousFunctionExpression
+//jscs:disable disallowMultipleLineBreaks
+//jscs:disable disallowSpacesInFunctionExpression

```

Comment:
Nice try ;-)

If you have a good reason to disable one of the checks I'm fine, but disabling all of them should be avoided.